Samples/BlazorExample/BlazorExample/Client/Shared/TextInput.razor

```diff
@@ -0,0 +1,12 @@
+﻿<div>
+  <input @bind-value="Property.Value" @bind-value:event="oninput"
```

Comment:
It'll be interesting to see how oninput performs when using server-side Blazor and the connection is across the internet. I haven't had a server to deploy on and test this yet.

Reply:
All this work is using server-side Blazor, but on my local network, so I don't really have a sense for performance. Michael Washington has been doing a lot of SSB over the Internet though, and says it is amazingly fast and responsive.